### ISSUE TITLE
Side specific input/output not working in most cases

### ISSUE THREAD

**Issue by GAsplund** [TeamDman/SuperFactoryManager#21]
Top on receiving inventory is set to input. Works fine using Ender IO item conduits, does not work using same inventory as input using SFM to output to top of same machine.

Tested and working to output to other inventories, but machines tested in question (Thermal Expansion Pulverizer, Redstone Furnace and Industrial Foregoing Black Hole Unit) will not receive nor send items.

(Interesting behavior on Black Hole Unit is that it will only receive items when its inventory is empty, but always receives from conduits.)

**Comment by Kiefy** [comment]
I think I just discovered the same issue with Applied Energistics 2 Inscribers.  Inserting items is fine, but I can't get it to detect or extract the printed circuits.

I know the Inscriber is a bit finicky about which sides it wants to insert/extract from. AE2's Wiki states the output can be automated from any side, but just in case, I tried physically connecting the inventory cable to the right and back sides of the Inscribers while enabling all sides within SFM gui.  Neither the Condition node, nor the In/Out nodes will acknowledge the existence of the output items.
